manifests/Inkscape/Inkscape/1.0.1.yaml

```
@@ -0,0 +1,15 @@
+Id: Inkscape.Inkscape
+Version: 1.0.1
+Name: Inkscape
+Publisher: The Inkscape Project
+License: GPL 3.0 or later
+LicenseUrl: https://inkscape.org/about/license/
+AppMoniker: inkscape
+Tags: inkscape, editor, foss, cross-platform, svg, vector-graphics, icons, graphics, export, drawing, art
+Description: Professional vector graphics editor
+Homepage: https://inkscape.org/
+Installers:
+  - Arch: x64
+    Url: https://inkscape.org/gallery/item/21592/inkscape-1.0.1-x64.msi
```

```suggestion
    Url: https://media.inkscape.org/dl/resources/file/inkscape-1.0.1-x64.msi
```
This is the direct link to the installer.

---

Ok, thanks! I'll try it.